web-client/src/presenter/sequences/batchDownloadDocketEntriesSequence.ts

```diff
@@ -0,0 +1,21 @@
+import { addPrintableDocketRecordCheckAction } from '@web-client/presenter/actions/addPrintableDocketRecordCheckAction';
+import { batchDownloadDocketEntriesAction } from '@web-client/presenter/actions/batchDownloadDocketEntriesAction';
+import { clearModalAction } from '../actions/clearModalAction';
+import { generateDocketRecordPdfUrlAction } from '@web-client/presenter/actions/generateDocketRecordPdfUrlAction';
+import { setShowModalAction } from '../actions/setShowModalAction';
+
+export const batchDownloadDocketEntriesSequence = [
+  clearModalAction,
+  addPrintableDocketRecordCheckAction,
+  {
```

Comment:
Nit: I'm not sure we need an entire cerebral condition to check this variable: we could combine `addPrintableDocketRecordCheckAction` and `generateDocketRecordPdfUrlAction`